mathics/builtin/system.py

```diff
@@ -347,6 +350,24 @@ def evaluate(self, evaluation):
         return Integer(size << 1)
 
 
+class UserName(Predefined):
+    """
+    <dl>
+      <dt>$UserName
+      <dd>returns a string describing the type of computer system on which
+      \Mathics is being run.
+    </dl>
+
+    X> $UserName
```

Comment:
This is not suitable for CI. The test should be implemented in pytest

Reply:
I am really confused about the meaning of `X>` and how this all works....

Here was the intent...

I want something that *isn't* run in doctest, but appears as an example.  How is this coded.

```
X> $UserName
= rocky
```

or 

```
X> $UserName
= ...
```

or 
``` 
X> $UserName
```

?

In our docs where we explain this stuff, `>>`, `X>`, `#>`, `=`, `...`  it woudl be great if we described this, and gave examples where one is appropriate and why.

Reply:
I'm confused about that too. I didn't realize those where not mentioned in the documentation. I'll try to fix this when I'm refactoring the wiki.

Reply:
Ok. Thanks for fixing when you get a chance. 

What should be used when youwant something that isn't run in doctest, but does appear as an example -  and with some output that is typical of output that might be produced?